Documentation/RELEASE_NOTES.md

```diff
@@ -15,6 +15,7 @@
 * Added multi cpu core support for NSGV (METROAE-309)
 * Improve VSD security hardening to monitor service shutdown (METROAE-285)
 * Add next-hop to VSC VPRN config (METROAE-319)
+* Add VCIN SA upgrade support for Major Minor and Inplace Upgrade(METROAE-317)
```

Comment:
Please add a set of menu items for vcin upgrade. (i.e. "metroae upgrade vcins inplace")

Reply:
Done